docs/maintenance/Team.mdx

```diff
@@ -0,0 +1,98 @@
+---
+id: team
+title: Team
+---
+
+import { TeamBioDivider } from '@site/src/components/team/TeamBioDivider';
+import { TeamBioList } from '@site/src/components/team/TeamBioList';
+
+The typescript-eslint organization is an independent open source project. Its
+members are volunteer open source developers dedicated to making the
+TypeScript linting experience great.
+
+<TeamBioDivider />
+
+## Maintainers
+
+<TeamBioList
+  bios={[
+    {
+      description:
+        'Computer scientist. Cider drinker. Linting enthusiast. Australian.',
+      links: [
+        ['twitter', 'https://twitter.com/bradzacher'],
+        ['mastodon', 'https://mastodon.online/@bradzacher'],
```

Comment:
I'm more active on threads.net but they are still testing support for ActivityPub 😢